sdk/va-report-components/documentation/docs/guides/esm.md

```diff
@@ -0,0 +1,127 @@
+---
+id: esm
+title: ES Module Imports
+---
+
+Sites built using a code bundler may choose to load `@sassoftware/va-report-components` with ES module imports. When loaded this way, the APIs are available as named imports.
+
+[SASReportElement](api/SASReportElement.md) and the other custom elements are defined through side effects, so care must be taken to ensure `@sassoftware/va-report-components` is included at runtime. One way to do this is by adding a side effect import in the build's entry point or in the modules where `va-report-components` is used.
+
+```js
+import "@sassoftware/va-report-components";
+```
+
+A side effect import might not be necessary if the code uses an imported API directly.
+
+```js
+import { SASReportElement } from "@sassoftware/va-report-components";
+const report = new SASReportElement();
+document.body.appendChild(report);
+```
+
+For TypeScript applications, type imports are insufficent for ensuring that `va-report-components` will be included in the build.
+
+```ts
+// Type imports are NOT included at runtime
+import type { SASReportElement } from "@sassoftware/va-report-components";
+// This report will not load unless va-report-components is imported elsewhere
+const report = document.getElementById("my-report") as SASReportElement;
+```
+
+## Usage notes
+
+### Webpack
+
+#### Configuration
+
+`va-report-components` contains CSS files that might not work as intended when processed by Webpack loaders. If your build uses loaders such as style-loader or css-loader, you should exclude `va-report-components`.
+
+```js
+// webpack.config.js
+export default {
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ["style-loader", "css-loader"],
+        exclude: /va-report-components/,
+      },
+    ],
+  },
+};
+```
+
+#### SAS Report Packages
+
+If your site hosts [exported SAS Report Packages](guides/export-report-package.md) alongside your webpack build, they may be referenced relative to `__webpack_public_path__`.
```

Comment:
Should webpack have an initial cap? On line 37 above, it has an an initial cap. I'm not sure which is correct.

Reply:
Thanks for bringing this inconsistency. Webpack is always lowercase in the tool's documentation except for at the start of sentences.